src/Model/Item.php

```diff
@@ -4265,4 +4265,22 @@ public static function getQuoteUriId(string $body, int $uid = 0): int
 		Logger::warning('Post does not exist although it was supposed to had been fetched.', ['id' => $id, 'url' => $url, 'uid' => $uid]);
 		return 0;
 	}
+
+	public static function countInbound(string $network)
```

Comment:
Since this method increase the count by 1, it should be called accordingly.
```suggestion
	public static function incrementInbound(string $network)
```

Reply:
Done